server/services/mi-light/lib/light/index.js

```diff
@@ -0,0 +1,31 @@
+const { configureBridge } = require('./light.configureBridge');
+const { getBridges } = require('./light.getBridges');
+const { getZones } = require('./light.getZones');
+const { setValue } = require('./light.setValue');
+const { init } = require('./light.init');
+
+/**
+ * @description Add ability to control a MiLight light
+ * @param {Object} gladys - Gladys instance.
+ * @param {Object} milightClient - Mi Light Client.
+ * @param {string} serviceId - UUID of the service in DB.
+ * @example
+ * const exampleLightHandler = new ExampleLightHandler(gladys, client, serviceId);
+ */
+const MiLightLightHandler = function MiLightLightHandler(gladys, milightClient, serviceId) {
+  this.gladys = gladys;
+  this.milightClient = milightClient;
+  this.serviceId = serviceId;
+  this.bridges = [];
+  this.connectedBridges = [];
+  this.bridgesByMac = new Map();
```

Comment:
Are you sure to need a Map object ? Simple (JSON) object is not enough ?

Reply:
Is it a big deal ? Because I'm not sure to understand the difference between these two object types.

Reply:
@Pierre-Gilles any opinion ?

Reply:
Map is really great! We should really use more of them, it's way better and safer that simple JSON objects, because it provides safe methods to check properties existence for ex.